guides/common/modules/proc_configuring-external-dns.adoc

```diff
@@ -46,6 +46,8 @@ endif::[]
 # chown -v root:named /etc/rndc.key
 # chmod -v 640 /etc/rndc.key
 ----
++
+Ensure the `foreman-proxy` user can read `/etc/rndc.key`.
```

Comment:
This is redundant and why the block on line 65 is there.

Reply:
I had information that this is not sufficient.

Reply:
I certainly think this is the wrong place but I do think the two blocks can be merged. Make it roughly something like
> To ensure the `foreman-proxy` user can read `/etc/rndc.key`, assign the `foreman-proxy` user ...

This makes it clear to any reader what the purpose of the action is which can help when something doesn't work.